### PR TITLE
add model

### PR DESCRIPTION
**Pull request by FreeBreadsticks**

### CODE REVIEW

File: app/models/test.rb [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#this is a test
```
Comment (gruzzlymug) [inline app/models/test.rb:1]: Great comment!

File: test/fixtures/tests.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Read about fixtures at http://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
+
+# This model initially had no columns defined.  If you add columns to the
+# model remove the '{}' from the fixture names and add the columns immediately
+# below each fixture, per the syntax in the comments below
+#
+one: {}
+# column: value
+#
+two: {}
```
Comment (gruzzlymug) [inline test/fixtures/tests.yml:10]: Why did you do this?

Comment (FreeBreadsticks) [inline test/fixtures/tests.yml:10]: because it works

**Comment by gruzzlymug** [comment]
:+1:

**Pull request merged**